"Internal Server Error" in blocks page.

1. run tapyrusd
2. run electrs 
3. run backend
4. run frontend
5. access frontend

then, backend generates error like this:

```
[2020-12-06T18:00:26.234] [ERROR] default - Error retrieving 25 blocks for page#1. Error Message - Internal Server Error
```

Error detail is the followings:
```
RpcError: Internal Server Error
    at Parser.rpc (/Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus/tapyrus-explorer/backend/node_modules/bitcoin-core/dist/src/parser.js:73:13)
    at Client.command (/Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus/tapyrus-explorer/backend/node_modules/bitcoin-core/dist/src/index.js:169:24)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Promise.all (index 16)
    at async /Users/h_yamaguchi/Documents/Projects/chaintope/tapyrus/tapyrus-explorer/backend/actions/block_list.js:56:20 {
  body: 'Work queue depth exceeded',
  code: 500
}
```

Setting `rpcworkqueue` parameter in tapyrus.conf and restarting tapyrusd will solve this problem, but can i configure in tapyrus-explorer?

--- a/setup.md
+++ b/setup.md
@@ -4,8 +4,15 @@
 
 ### Tapyrusd
 
+The following settings need to be added to the configuration file:
+
+```
+rpcworkqueue=32
+rpcthreads=16
+```
+
 Run tapyrusd
-```sh
+```bash
 $ sudo ./src/tapyrusd -datadir=/var/lib/tapyrus-testnet -conf=/etc/tapyrus/tapyrus.conf 
 ```
 

